### PR TITLE
Fixing loop

### PR DESCRIPTION
**Pull request by timheuer**
When a language detection failed it was in an endless loop.

Also added devcontainer and check if env already exists.

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -20,7 +23,7 @@ export function activate(context: vscode.ExtensionContext) {
 		}
 
 		// Get all files in workspace, ignore .json files, node_modules and .md, .gitignore
```
Comment (savannahostrowski) [inline src/extension.ts]: Mind updating the comment?

File: src/ai.ts [hunk 2]
```diff
@@ -4,13 +4,16 @@ export async function chatGPTRequest(prompt: string) {
...
-    
+
     const openaiClient = new openai.OpenAIApi(configuration);
 
     try {
         const gptResponse = await openaiClient.createChatCompletion({
             model: "gpt-3.5-turbo",
-            messages: [{role: "user", content: prompt}],
+            messages: [
+                { role: "system", content: "You are a programming expert. You can detect software development languages based on a set of files that represent a folder of source code." },
```
Comment (savannahostrowski) [inline src/ai.ts:14]: Did you find that this yielded an improved result? I haven't found this to be necessary on my projects.

**Review by savannahostrowski — CHANGES_REQUESTED** [review]
Two small comments/questions but otherwise LGTM. Thanks for the PR.

File: src/ai.ts (hunk 2 above)
Comment (timheuer) [reply src/ai.ts:14]: Yes I did actually...on the C# one it failed to identify anything, adding this and it identified "C#, XML"

**Review by savannahostrowski — APPROVED** [review]

**Pull request merged**